core/src/CopyChip/CopyChip.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import cx from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClipboard, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Button } from "../Button";
+
+import styles from "./CopyChip.less";
+
+export const CopyChip = (props) => {
```

Comment:
If we have settled on this being a button lets not refer to it as a "Chip"

Reply:
swapped to CopyButton